modules/admin-ui-frontend/app/scripts/modules/events/partials/eventActionsCell.html

```diff
@@ -62,3 +62,9 @@
    with-role="ROLE_UI_EVENTS_DETAILS_ASSETS_VIEW">
 </a>
 
+<a ng-click="row.embeddingCode()"
+   class="embeddingcode"
+   title="{{ 'EVENTS.EVENTS.TABLE.TOOLTIP.EMBEDDING_CODE' | translate }}">
```

Comment:
We should display this only for users with related roles, something like the following

```
<a ng-click="row.embeddingCode()"
class="embeddingcode"
title="{{ 'EVENTS.EVENTS.TABLE.TOOLTIP.EMBEDDING_CODE' | translate }}"
with-role="ROLE_UI_EVENTS_EMBEDDING_CODE">
</a>
```